### PR TITLE
restructure cpu section

### PR DESCRIPTION
**Pull request by bellesea**

### CODE REVIEW

File: guide/cpu/cpu.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# The CPU
```
Comment (jessicard) [inline guide/cpu/cpu.md:1]: Do you think this section could be combined with the instruction cycle?

File: guide/cpu/physical-world.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Processor clock
+
+In our warehouse, we want to make sure everything happens at an organized pace, and there aren't any disruptions at stations. One way we can accomplish this is by setting everything to a timer. Let's say our boxes move forward at the pace of 1 station per second, and each station takes 1 second to perform its task. Back in CPU land, this would be our **processor clock**.
+
+It's not a clock that would be useful for you or me, but is made of material that oscillates, moving back and forth at a certain frequency, giving you a bunch of vibrations per second. We call each one of these vibrations a "clock tick". These vibrations helps keeps the CPU in sync as each "clock tick" signals the CPU starts a new instruction by fetching the instruction data from memory.
+
+This is important to us because for every clock tick, the CPU reads one instruction. This clock goes _really fast_. Today's CPUs are measured in gigahertz (GHz) which is approximately **one billion cycles per second**!
+
+<a href="/guide/cpu/instruction-cycle.md">
```
Comment (jessicard) [inline guide/cpu/physical-world.md:31]:
```suggestion
<br />

---

<a href="/guide/cpu/instruction-cycle.md">
```

For the footers, we're missing a little bit of markup here. Wish we had partials, lol.

File: guide/cpu/cpu.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+</p>
+<br />
+
+Have you heard of the companies Intel or AMD? These are two popular companies that manufacture the CPUs that go into our computers. All of the computers we use contain something called a central processing unit, also known as the CPU or the processor, which effectively acts as the brain of the computer.
+
+Computers contain other processing units (like the graphics card!) that are responsible for processing other more specific things, but the CPU is your general powerhouse for all computing tasks. That being said, the CPU can do shockingly little: it can **read values**, **set values**, and **perform simple math** calculations like addition and subtraction.
+
+You hand it numbers, and it’s put to work crunching the data however you’d like. That's it. Everything your computer doing is made up of just that. Isn't that wild?
+
+<a href="/guide/introduction.md">
```
Comment (jessicard) [inline guide/cpu/cpu.md:21]:
```suggestion
<br />

---

<a href="/guide/introduction.md">
```

If we keep this section separate, we'll need to add the footer fix here too.

**Review by jessicard — COMMENTED** [review]
We should also update the table of contents in this PR!

File: guide/cpu/cpu.md (hunk 1 above)
Comment (bellesea) [reply guide/cpu/cpu.md:1]: ooo yes! will make the changes.

File: guide/cpu/communicating.md [hunk 4]
```diff
@@ -1,33 +1,3 @@
...
-
-### Processor clock
-
-In our warehouse, we want to make sure everything happens at an organized pace, and there aren't any disruptions at stations. One way we can accomplish this is by setting everything to a timer. Let's say our boxes move forward at the pace of 1 station per second, and each station takes 1 second to perform its task. Back in CPU land, this would be our **processor clock**.
-
-It's not a clock that would be useful for you or me, but is made of material that oscillates, moving back and forth at a certain frequency, giving you a bunch of vibrations per second. We call each one of these vibrations a "clock tick". These vibrations helps keeps the CPU in sync as each "clock tick" signals the CPU starts a new instruction by fetching the instruction data from memory.
-
-This is important to us because for every clock tick, the CPU reads one instruction. This clock goes _really fast_. Today's CPUs are measured in gigahertz (GHz) which is approximately **one billion cycles per second**!
-
 ## Communicating with the CPU
```
Comment (jessicard) [inline guide/cpu/communicating.md:1]:
```suggestion
# Communicating with the CPU
```

Since it's top level now!

**Comment by bellesea** [comment]
> Last thing is to update the table of contents and then I think we're good to go!!

thank you!

**Comment by bellesea** [comment]
> Last thing is to update the table of contents and then I think we're good to go!!

done, thank you!

**Review by jessicard — APPROVED** [review]
Nice @bellesea! ⭐

**Pull request merged**